packages/kbn-esql-validation-autocomplete/src/autocomplete/autocomplete.test.ts

```diff
@@ -271,28 +271,20 @@ describe('autocomplete', () => {
           async (text) => (text ? getAstAndSyntaxErrors(text) : { ast: [], errors: [] }),
           callbackMocks
         );
-        const suggestionInertTextSorted = suggestions
-          // simulate the editor behaviour for sorting suggestions
-          // copied from https://github.com/microsoft/vscode/blob/0a141d23179c76c5771df25a43546d9d9b6ed71c/src/vs/workbench/contrib/testing/browser/testingDecorations.ts#L971-L972
-          // still not sure how accurate this is...
-          .sort((a, b) => (a.sortText || a.label).localeCompare(b.sortText || b.label));
 
-        expect(suggestionInertTextSorted).toHaveLength(expected.length);
-        for (const [index, receivedSuggestion] of suggestionInertTextSorted.entries()) {
-          if (typeof expected[index] !== 'object') {
-            expect(receivedSuggestion.text).toEqual(expected[index]);
-          } else {
-            // check all properties that are defined in the expected suggestion
-            for (const [key, value] of Object.entries(expected[index])) {
-              expect(receivedSuggestion[key as keyof SuggestionRawDefinition]).toEqual(value);
-            }
-          }
-        }
```

Comment:
Basically this logic was making life difficult while adding very little value.

The idea was to check the order of the generated suggestions as they would be shown in the editor. However, this isn't actually the logic that Monaco uses to sort these things and that made this basically an arbitrary check that verified one part of our own code against another part of our own code instead of against the true behavior of the editor.

At some point if we're able to truly decipher the sorting behavior of Monaco, I could see this being somewhat useful, but... even then... it feels like a true verification would actually involve Monaco itself in a browser. And, at the end of the day, will the world end if the suggestions show up in a slightly different order? Just doesn't feel worth it...

Open for other opinions though!